jlm/tooling/CommandLine.cpp

```diff
@@ -670,62 +694,102 @@ JlmOptCommandLineParser::ParseCommandLineArguments(int argc, char **argv)
 
   auto aASteensgaardAgnostic = JlmOptCommandLineOptions::OptimizationId::AASteensgaardAgnostic;
   auto aASteensgaardRegionAware = JlmOptCommandLineOptions::OptimizationId::AASteensgaardRegionAware;
-  auto commonNodeElimination = JlmOptCommandLineOptions::OptimizationId::cne;
-  auto deadNodeElimination = JlmOptCommandLineOptions::OptimizationId::dne;
-  auto functionInlining = JlmOptCommandLineOptions::OptimizationId::iln;
+  auto commonNodeElimination = JlmOptCommandLineOptions::OptimizationId::CommonNodeElimination;
+  auto commonNodeEliminationDeprecated = JlmOptCommandLineOptions::OptimizationId::cne;
+  auto deadNodeElimination = JlmOptCommandLineOptions::OptimizationId::DeadNodeElimination;
+  auto deadNodeEliminationDeprecated = JlmOptCommandLineOptions::OptimizationId::dne;
+  auto functionInlining = JlmOptCommandLineOptions::OptimizationId::FunctionInlining;
+  auto functionInliningDeprecated = JlmOptCommandLineOptions::OptimizationId::iln;
   auto invariantValueRedirection = JlmOptCommandLineOptions::OptimizationId::InvariantValueRedirection;
-  auto nodePushOut = JlmOptCommandLineOptions::OptimizationId::psh;
-  auto nodePullIn = JlmOptCommandLineOptions::OptimizationId::pll;
-  auto nodeReduction = JlmOptCommandLineOptions::OptimizationId::red;
-  auto thetaGammaInversion = JlmOptCommandLineOptions::OptimizationId::ivt;
-  auto loopUnrolling = JlmOptCommandLineOptions::OptimizationId::url;
+  auto nodePushOut = JlmOptCommandLineOptions::OptimizationId::NodePushOut;
+  auto nodePushOutDeprecated = JlmOptCommandLineOptions::OptimizationId::psh;
+  auto nodePullIn = JlmOptCommandLineOptions::OptimizationId::NodePullIn;
+  auto nodePullInDeprecated = JlmOptCommandLineOptions::OptimizationId::pll;
+  auto nodeReduction = JlmOptCommandLineOptions::OptimizationId::NodeReduction;
+  auto nodeReductionDeprecated = JlmOptCommandLineOptions::OptimizationId::red;
+  auto thetaGammaInversion = JlmOptCommandLineOptions::OptimizationId::ThetaGammaInversion;
+  auto thetaGammaInversionDeprecated = JlmOptCommandLineOptions::OptimizationId::ivt;
+  auto loopUnrolling = JlmOptCommandLineOptions::OptimizationId::LoopUnrolling;
+  auto loopUnrollingDeprecated = JlmOptCommandLineOptions::OptimizationId::url;
 
   cl::list<JlmOptCommandLineOptions::OptimizationId> optimizationIds(
     cl::values(
       ::clEnumValN(
         aASteensgaardAgnostic,
         JlmOptCommandLineOptions::ToCommandLineArgument(aASteensgaardAgnostic),
-        "Steensgaard alias analysis with agnostic memory state encoding."),
+        "Steensgaard alias analysis with agnostic memory state encoding"),
       ::clEnumValN(
         aASteensgaardRegionAware,
         JlmOptCommandLineOptions::ToCommandLineArgument(aASteensgaardRegionAware),
-        "Steensgaard alias analysis with region-aware memory state encoding."),
+        "Steensgaard alias analysis with region-aware memory state encoding"),
```

Comment:
Since the other are with capital letters, should it not be Steensgaard Alias Analysis